clients/da-vinci-client/src/main/java/com/linkedin/davinci/storage/StorageService.java

```diff
@@ -371,6 +375,38 @@ public synchronized AbstractStorageEngine openStore(
     return engine;
   }
 
+  public synchronized void checkWhetherStoragePartitionsShouldBeKeptOrNot(SafeHelixManager manager) {
+    if (getStorageEngineRepository() == null || manager == null) {
```

Comment:
nit: getStorageEngineRepository() == null isn't needed.  The getter is based off of a final member variable.